docs/UserGuide.md

```diff
@@ -71,23 +71,37 @@ Cautions are placed in this guide to serve as warnings for certain actions.
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [s/SKILLS]` can be used as `n/John Doe s/Python` or as `n/John Doe`.
+* The command word and the item type for the command are case-sensitive. They have to be `lower_case`. <br>
+  e.g. in `edit 1 n/NAME`, the command word `add` is valid as it is in
+  `lower_case` and `n/` has to be `lower_case` as well.<br>
+  That is, `Add n/John Doe...` will be an invalid command because the command word `Add` is not `lower_case`.
+
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. in `add n/NAME p/PHONE_NUMBER e/EMAIL j/JOB r/ROUND [s/SKILL]...`.<br> 
```

Comment:
Seems a bit long, probably can just do `sort f/NAME ...` or `edit` as a shorter example to cut length as the features probably have examples as well? Same for other points.